### ISSUE TITLE
Android 13 permissions compatibility

### ISSUE THREAD

**Issue by daniel-waiguru** [Dhaval2404/ImagePicker#318]
## Summary

Getting "allow access photos and media n this device error" on android 13, yet I have allowed the permission; see below screenshots
![Screenshot_20230117_113131_iPOS Owner](https://user-images.githubusercontent.com/52042764/212848914-8afec56e-d7f9-4d24-940b-16372a7354f0.jpg)
![Screenshot_20230117_113158_Settings](https://user-images.githubusercontent.com/52042764/212848968-c0d2ecfd-020f-4eed-821f-190f8c1ebf6a.jpg)

## Android version

13

## Impacted devices

Samsung

**Comment by ArpitSharma32** [comment]
yes I am also facing the same error, is there any quick work around for the same ?
